src/emqx_ct_helpers_jobs.erl

```diff
@@ -20,8 +20,8 @@
          is_job_node/0, node_name/1, ct_master_orchestration/3, repo_root/0, container_ip/1,
          container_ip/2, test_platform_host/0]).
 
--include("./emqx_ct_job.hrl").
--include("./emqx_ct_jobs_transform.hrl").
```

Comment:
I’m not even sure why I had “./” in the includes, might have been the IDE refactor ( rename ). Thanks for cleaning it